.gitignore

```diff
@@ -13,7 +13,9 @@ proxyui/plugins/
 h2proxydb/
 proxyui/seleniumSslSupport/
 proxyui/backup-uploaded.json
+proxyui/last-restored-db.zip
 proxyui/tomcat.ks
 proxyui/tmp
+proxyui/null
```

Comment:
why null ?

Reply:
`getPath` is returning this path, I think some variable setup is missing in odo. Anyway it's not a big problem, it's kind of a temporary path for the backup that will be downloaded.